Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -218,6 +226,14 @@ private function mapIncludeFilter(StoreInterface $store, IncludeFilters &$includ
             }
             $this->mapValueToFilter($includeFilters, $store, $attributeCode, $value);
         } catch (NostoException $e) {
+            $this->logger->debugWithSource(
```

Comment:
These logging statements seem bloated. The logger erasure should be cleaned up. We have 5-7 lines for one log statement and this is drastically reducing the readability of the code.